misk/src/test/kotlin/misk/config/ConfigTest.kt

```diff
@@ -24,6 +24,7 @@ class ConfigTest {
   val module = Modules.combine(
       ConfigModule.create("test_app", config),
       EnvironmentModule(defaultEnv)
+//    @TODO(jwilson swankjesse) https://github.com/square/misk/issues/272 breaks here
```

Comment:
nit: typically for TODO comments we indent at the right level (so `//` is beneath `Environment) and you can chose one username for me. Lets go with jwilson for this.